src/interactivity/win32/uiaTextRange.cpp

```diff
@@ -71,12 +71,16 @@ IFACEMETHODIMP UiaTextRange::Clone(_Outptr_result_maybenull_ ITextRangeProvider*
 
 void UiaTextRange::_TranslatePointToScreen(til::point& clientPoint) const
 {
-    ClientToScreen(_getWindowHandle(), clientPoint.as_win32_point());
+    auto win32Point = clientPoint.as_win32_point();
+    ClientToScreen(_getWindowHandle(), win32Point);
+    clientPoint = til::point{ *win32Point };

```

Comment:
honestly I'd revert this. It's actually not doing what you think - since you have a reference, and get a pointer to it, `ClientToScreen` is _still_ directly overwriting `clientPoint`. Better to skip the middleman and stick with the code before this commit. :D